CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+Welcome!
+Every person creating InnerSource-related content or tools within their company should contribute to this working group.
+This working group helps all to get feedback on this work and to share it so that others can leverage it as well.
+Check out our [README.md] for more details.
+
+## 3 Ways to Contribute
+
+1. Contribute your InnerSource-related content.
+
+    * We track contributions on this [Kanban board].
+    * Get access to contribute to the board by [joining the ISPO Working Group team].
+    * Once you have access, create a new card in the **No Status** column describing the content that you want to contribute and what challenge it helps to solve.
```

Comment:
Easier to read with:

```suggestion
Create a new card in the **No Status** column describing your contribution and the challenge it helps solve.
```

Do we need to add _Once you have access_ or the content, as both are clearly implied?